robozonky-app/src/main/assembly/resources/examples/services/robozonky-systemd.service

```diff
@@ -3,9 +3,15 @@ Description=RoboZonky: Automated Zonky.cz investing robot
 After=network.target
 
 [Service]
-Restart=true
+User=robozonky
+Group=robozonky
+Restart=always
 WorkingDirectory=/usr/robozonky
-ExecStart=/usr/robozonky/robozonky.sh @robozonky.cli
+Environment=\
```

Comment:
Viz můj komentář - asi bych byl pro, nechat to tak, jak to bylo. Je nějaký důvod, proč je potřeba to změnit?